### PR TITLE
1194899 locales api

### PR DESCRIPTION
**Pull request by willkg**
This is pretty straight-forward--nothing exciting here. Quick r?

### CODE REVIEW

File: docs/api.rst [hunk 1]
```diff
@@ -144,3 +144,51 @@ Using curl::
...
+            "name": "English",
+            "localized_name": "English",
+            "aaq_enabled": true
+         },
+         ...
+      }
+
+   :reqheader Accept: application/json
+   :resheader Content-Type: application/json
+   :statuscode 200: no error
```
Comment (willkg) [inline docs/api.rst]: @mythmon What do you think about documenting apis this way? If we ever get around to building autodocumenting apis, I think I'd want to use this format. It's from here:

http://pythonhosted.org/sphinxcontrib-httpdomain/

File: kitsune/sumo/api.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        # FIXME: should we add something based on SIMPLE_WIKI_LANGUAGES?
+        locale = {
+            'name': LOCALES[lang].english,
+            'localized_name': LOCALES[lang].native,
+        }
+        try:
+            QuestionLocale.objects.get(locale=lang)
+            locale['aaq_enabled'] = True
+        except QuestionLocale.DoesNotExist:
+            locale['aaq_enabled'] = False
```
Comment (willkg) [inline kitsune/sumo/api.py]: I added the bits you specified in the bug. Do we want to add something for `SIMPLE_WIKI_LANGUAGES`? Are there other things we want to add?

File: docs/api.rst (hunk 1 above)
Comment (mythmon) [inline docs/api.rst]: I like `sphinxcontrib.httpdomain`. I think it is a good idea to use something already made for documenting APIs. +1

Also, thanks so much for writing API docs here. You're an inspiration to us all.

File: kitsune/sumo/api.py (hunk 2 above)
Comment (mythmon) [inline kitsune/sumo/api.py]: SIMPLE_WIKI_LANGUAGES could be useful. I don't know of any other properties we need, but if you have any other ideas, I think that would be fine.

Comment (mythmon) [inline kitsune/sumo/api.py]: On another note, couldn't this stanza be simplified to `locale['aaq_enabled'] = QuestionLocale.objects.filter(locale=lang).exists()`? (Or am I making that up?)

Comment (willkg) [inline kitsune/sumo/api.py]: I don't think I've ever used `.exists()`. I can test that now.

**Comment by willkg** [comment]
^^^ That removes the FIXME and also tweaks the code to use `.exists()`.

**Comment by mythmon** [comment]
Looks good!

**Comment by willkg** [comment]
Thank you!

**Pull request merged**